src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossDF.scala

```diff
@@ -56,7 +57,10 @@ object TreeLossDF {
 
   }
 
-  def primaryForestFilter(include: Boolean)(df: DataFrame): DataFrame = {
+  def contextualLayerFilter(
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6100986)